CIP-0021/README.md

```diff
@@ -104,6 +108,10 @@ In this exceptional case, auxiliary data must be encoded in their "tuple" format
 
 The `auxiliary_scripts` must be an array of length 0.
 
+**Network ID**
+
+Since Alonzo era `network_id` must always be included in the transaction body.
```

Comment:
This is ambiguous, does not make it clear that network id must be included in pre-Alonzo txs as well.